packages/frontend/src/services/StakingFarmContracts.js

```diff
@@ -46,4 +60,34 @@ export default class StakingFarmContracts {
             )
         );
     }
+
+    static isFarmingValidator(accountId) {
+        return (
+            getValidationVersion(
+                NODE_URL.indexOf(MAINNET) > -1 ? MAINNET : TESTNET,
+                accountId
+            ) === FARMING_VALIDATOR_VERSION
+        );
+    }
+
+    static hasUnclaimedRewards = async ({
+        contractName,
+        account_id,
+        from_index,
+        limit,
```

Comment:
```suggestion
        from_index = 0,
        limit = 300,
```
Are these likely to change per use-case? Should we default them here instead of passing them at the call-site?